File: Client/Scenes/NetComClasses/Stats.cs
using System.Collections.Generic;

namespace Client.Scenes.NetComClasses
{
    public class Stats
    {
        public Stats(float hp, int ap, int mp, int spice, bool isLoud, bool isSwallowed)
        {
            HP = hp;
            AP = ap;
            MP = mp;
            this.spice = spice;
            this.isLoud = isLoud;
            this.isSwallowed = isSwallowed;
        }

        public float HP { get; set; }
        public int AP{ get; set; }
        public int MP { get; set; }
        public int spice { get; set; }
        public bool isLoud { get; set; }
        public bool isSwallowed { get; set; }
    }
}

File: KI/KI-Client/src/main/java/messages/data/dictionary/Stats.java
package messages.data.dictionary;

/**
 * This class represents stats for characters used in the message format.
 */
public class Stats {

  public final int HP;
  public final int AP;
  public final int MP;
  public final int spice;
  public final boolean isLoud;
  public final boolean isSwallowed;

  public Stats(final int HP, final int AP, final int MP, final int spice, final boolean isLoud,
      final boolean isSwallowed) {
    this.HP = HP;
    this.AP = AP;
    this.MP = MP;
    this.spice = spice;
    this.isLoud = isLoud;
    this.isSwallowed = isSwallowed;
  }

  @Override
  public String toString() {
    return "{\n"
        + "HP: " + HP + ",\n"
        + "AP: " + AP + ",\n"
        + "MP: " + MP + ",\n"
        + "spice: " + spice + ",\n"
        + "isLoud: " + isLoud + ",\n"
        + "isSwallowed: " + isSwallowed + "\n"
        + "}";
  }
}
